src/CheatSheet.stories.mdx

```diff
@@ -0,0 +1,999 @@
+
+<MenuLink items={[
+  {
+    label: 'Vanilla',
+    id: 'vanilla',
+    disabled: true,
+  },
+  { label: 'Chocolate', id: 'chocolate' },
+]}/>
+</div>
+
+<div class="layout">
+<h4>
+<a href="/docs/components-interactions-checkbox--regular">Checkbox</a>
+</h4>
+<p>Checkbox allow users to select one or more options from a list. Checkboxes are typically used in situations where there are multiple options to choose from, and the user can select one or more options.</p>
+<div>
+  <Checkbox
+    checked
+    data-testid="checkbox-1"
+    defaultChecked
+    displayStyle="inline"
+    id="myInput1"
+    invalidText="A valid value is required"
+    onChange={() => {}}
+    value="1"
+  >
+    One
+  </Checkbox>
+</div>
+
+<h4>
+  <a href="/docs/components-interactions-checkbox--indeterminate">
+    Indeterminate
+  </a>
+</h4>
+
+<div>
+  <Checkbox
+    checked
+    data-testid="checkbox-1"
+    defaultChecked
+    displayStyle="inline"
+    id="myInput1"
+    indeterminate
+    invalidText="A valid value is required"
+    onChange={() => {}}
+    value="1"
+  >
+    One
+  </Checkbox>
+</div>
+
+<h4>
+<a href="/docs/components-interactions-checkbox--list-of-checkboxes">List of Checkbox</a>
+</h4>
+<div>
+  <div>
+    <Checkbox
+      checked
+      data-testid="checkbox-1"
+      defaultChecked
+      displayStyle="inline"
+      id="myInput1"
+      invalidText="A valid value is required"
+      onChange={() => {}}
+      value="1"
+    >
+      One
+    </Checkbox>
+  </div>
+  <div>
+    <Checkbox
+      checked
+      data-testid="checkbox-2"
+      defaultChecked
+      displayStyle="inline"
+      id="myInput2"
+      invalidText="A valid value is required"
+      onChange={function noRefCheck() {}}
+      value="2"
+    >
+      Two
+    </Checkbox>
+  </div>
+  <div>
+    <Checkbox
+      checked
+      data-testid="checkbox-3"
+      defaultChecked
+      displayStyle="inline"
+      id="myInput3"
+      invalidText="A valid value is required"
+      onChange={function noRefCheck() {}}
+      value="3"
+    >
+      Three
+    </Checkbox>
+  </div>
+  <div>
+    <Checkbox
+      checked
+      data-testid="checkbox-4"
+      defaultChecked
+      displayStyle="inline"
+      id="myInput4"
+      invalidText="A valid value is required"
+      onChange={function noRefCheck() {}}
+      value="4"
+    >
+      Four
+    </Checkbox>
+  </div>
+  <div>
+    <Checkbox
+      checked
+      data-testid="checkbox-5"
+      defaultChecked
+      displayStyle="inline"
+      id="myInput5"
+      invalidText="A valid value is required"
+      onChange={function noRefCheck() {}}
+      value="5"
+    >
+      Five
+    </Checkbox>
+  </div>
+</div>
+</div>
+
+<div class="layout">
+<h4>
+<a href="/docs/components-interactions-input--regular">Input</a>
+</h4>
+<p>Input allows users to enter data into a web app. It can be used in various ways to allow users to provide text, numbers, and other types of input. </p>
```

Comment:
"Inputs allow users to enter various types of text, numbers, or other types of input"